robolectric-annotations/build.gradle

```diff
@@ -0,0 +1,5 @@
+dependencies {
+    // Compile dependencies
+    provided "com.intellij:annotations:12.0"
+    provided "org.robolectric:android-all:6.0.0_r1-robolectric-0"
```

Comment:
android-all are the full aosp jars. The android.jar that comes with the SDK is what we should use to build against for everything except the framework shadows which need access to the internals of the AOSP jar. Everything else should just be using the android apis only. Think this helps us to not leak the internals and be closer to real work projects as users projects will build against the SDK android.jar also.